ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/TransactionValidator.java

```diff
@@ -58,6 +58,8 @@
     EXCEEDS_BLOCK_GAS_LIMIT,
     TX_SENDER_NOT_AUTHORIZED,
     CHAIN_HEAD_WORLD_STATE_NOT_AVAILABLE,
-    PRIVATE_TRANSACTION_FAILED
+    PRIVATE_TRANSACTION_FAILED,
+    PRIVATE_NONCE_TOO_LOW,
+    INCORRECT_PRIVATE_NONCE
   }
```

Comment:
Will you add a comment here and in JsonRpcErrorConverter indicating that they probably need to be updated together.